### ISSUE TITLE
Support MCNP importance tagging in UWUW workflow

### ISSUE THREAD

**Issue by gonuke** [svalinn/DAGMC#405]
Changes are probably isolated largely in `write_lcad_uwuw` and can be modeled on other tags.

Note: default importance should be 1 for any unset importances.

**Comment by makeclean** [comment]
Should likely be of the form
 
      Importance:ParticleName/Value

**Comment by gonuke** [comment]
Are UWUW names case-sensitive?  I would prefer if the weren't.  Is that another issues?

**Comment by makeclean** [comment]
Yes, through a very specific design choice. I wanted the workflow to be specific and exact with minimal likelihood of ambiguity. Every non match would be a failure, certainly for the values of keys, I would be happy for the keys to be non case sensitive, but I would be very against the values being case sensitive.

**Comment by gonuke** [comment]
you mean "against the values being **NON** case sensitive"?

**Comment by makeclean** [comment]
Sorry. Ugh. Yes, I would be against values being non case sensitive. I.e. I have a strong preference for case sensitivity in the values.
